src/test/java/org/aksw/simba/squirrel/fetcher/manage/FetcherDummyTest.java

```diff
@@ -0,0 +1,61 @@
+package org.aksw.simba.squirrel.fetcher.manage;
+
+import org.aksw.simba.squirrel.data.uri.CrawleableUri;
+import org.aksw.simba.squirrel.fetcher.Fetcher;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.File;
+import java.io.IOException;
+
+public class FetcherDummyTest implements Fetcher {
```

Comment:
A better naming would be `FetcherDummy4Tests` or something similar. At the moment, the name sounds like the class would be a test but it is a dummy (or muckup) class.

Reply:
@MichaelRoeder Changed the Class name to FetcherDummy4tests from FetcherDummytest.